api/controllers/itemController.js

```diff
@@ -0,0 +1,22 @@
+const {itemService} = require("../services");
+const {catchAsync} = require("../utils/error");
+
+const teabags = catchAsync(async (req, res) => {
+    const userId = req.user.id
+    const {category} = req.params;
+    let {order, tasting_notes} = req.query
+    let price = req.query.price
```

Comment:
```suggestion
    const userId = req.user.id
    const {category} = req.params;
    const {sorting, tasting_notes, price} = req.query
```
1. 가급적이면 const로 선언해주세요.  
2. price 또한 req.query에 있으므로 윗줄과 함께 한번에 구조분해할당이 가능하겠습니다.
3. order 라는 키워드는 주문과 연관있어보이므로, sorting과 같은 name으로 query를 받고 처리하는것이 명확할 것 같습니다.